src/Microsoft.Extensions.Logging/LoggerFactory.cs

```diff
@@ -127,7 +152,15 @@ public void AddProvider(string providerName, ILoggerProvider provider)
             }
         }
 
-        public void AddFilter(string providerName, string categoryName, Func<LogLevel, bool> filter)
+        /// <summary>
+        /// Adds a filter that applies to <paramref name="providerName"/> and <paramref name="categoryName"/> with the given
+        /// <paramref name="filter"/>, returning true means allow log through, false means reject log.
```

Comment:
Remove the `, returning ...` section because it's just describing the parameter. Put that text in the description of how ti implement the `filter` parameter.